### ISSUE TITLE
`v1.4.1` breaks lobby display

### ISSUE THREAD

**Issue by malsobay** [empiricaly/empirica#339]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

The upgrade from `v1.4.0` to `v1.4.1` introduces a bug where the lobby does not render -- games can start once lobbies are filled, so the players are being enrolled in the lobby, but all they see is the loading screen (with the loading circle animation) screenshotted below.

![image](https://github.com/empiricaly/empirica/assets/79336354/0d9340b6-6495-4387-8a91-4aa61ab2531c)

### Steps To Reproduce

1. With v1.4.1 of Empirica, create a new project
2. Launch Empirica within the new project, then create and start a batch with the default two player demo 
3. Complete the pre-lobby steps 
4. Rather than transitioning to a screen showing the "Waiting for other players message" (or any lobby given to the `EmpiricaContext`), the players are shown a loading animation. 

### Empirica Version

```text
Version: v1.4.1
SHA:     6a820e7
Branch:  main
Time:    2023-07-01T11:56:14Z

Client:  1.4.1
Server:  1.4.1
```

### What OS are you seeing the problem on?

macOS, Linux

### What browser are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by npaton** [comment]
This should be fixed in the latest (1.4.2).

**Comment by malsobay** [comment]
Seems to be working -- thanks!
